core/jobs_test.py

```diff
@@ -45,9 +51,7 @@ def _run(cls, additional_job_params):
 
 
 class MockJobManagerTwo(jobs.BaseDeferredJobManager):
-    @classmethod
-    def _run(cls, additional_job_params):
-        return 'output'
```

Comment:
This function was not doing anything and was not covered. _run need not be re-implemented here